var/spack/repos/builtin/packages/pacifica-cli/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PacificaCli(PythonPackage):
+    """Python CLI for Pacifica Core Services"""
+
+    homepage = "https://github.com/pacifica/pacifica-cli/"
+    url = "https://pypi.io/packages/source/p/pacifica-cli/pacifica-cli-0.5.2.tar.gz"
```

Comment:
```suggestion
    pypi = "pacifica-cli/pacifica-cli-0.5.2.tar.gz"
```
Same for other packages

Reply:
yes, good plan, but: ```==> Error: Package PacificaCli has no version with a URL.``` when I do that on our production spack checkout, I need this to merge, and work on our current 15.4 versions.

Reply:
Yes, that's one of the reasons why Spack's core and package repositories are currently in the same git repo. You could keep it as is, but someone like me may come along and update it to the new syntax. Could you update it in this PR but keep it the same on your local production environment?

Reply:
Sure thing, i'll need to do some merges to appropriate branches..

Reply:
Changes sent